src/components/Search/InstantSearch.tsx

```diff
@@ -38,7 +38,7 @@ const placeHolderCSS = css`
   font-weight: 500;
 `;
 
-const searchWrapper = (theme: RIDITheme) => css`
+const SearchWrapper = styled.div<{theme: RIDITheme}>`
```

Comment:
```suggestion
const SearchWrapper = styled.div<{}, RIDITheme>`
```

`styled` 컴포넌트가 알아서 theme을 가져옵니다.

Reply:
오 이런방법이..